src/style.css

```diff
@@ -217,3 +227,165 @@ button[disabled] {
 #board .player-on-goal {
   background: url("assets/player_on_goal.png");
 }
+
+#status {
+  margin: 20px;
+  letter-spacing: 2px;
+  text-align: center;
+  width: 300px;
+
+  animation: fadeInDown ease 1s both;
+}
+
+#status-box {
+  word-wrap: break-word;
+}
+
+/* Collapsable Contents */
+#stats {
+  font-size: 12px;
+  padding: 0 18px;
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height 0.2s ease-out;
+}
+
+#collapse {
+  content: expand_circle_down;
+  color: #00dfef;
+  cursor: pointer;
+  border: none;
+  text-align: left;
+  outline: none;
+  font-size: 15px;
+  height: 20px;
+  transition: 0.4s;
+}
+
+#collapse:after {
+  font-weight: bold;
+  float: right;
+  margin-left: 5px;
+}
+
+
+
+/* The Modal (background) */
+#modal {
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; 
+  height: 100%;
+  overflow:disabled; /* Disable scroll */
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0,0.65); /* Black w/ opacity */
+}
+
+.text-glow {
+  font-size: 20px;
+  text-shadow:
+    /* outer glow */
+    0 0 10px #f0f,
+    0 0 20px #f0f,
+    0 0 20px #f0f;
+}
+
+.score {
+  padding-left:30px;
+}
+
+/* Modal Content Box */
```

Comment:
Usually modals can be closed by clicking on the darkened overlay around the modal, but that doesn't work with ours. It's a bit annoying to have to hunt for the "X" and click that specific area.

Using a simple modal library like https://github.com/idurar/vanilla-js-modal/tree/master/src probably saves some guesswork here?